blaze/server/spider.py

```diff
@@ -164,6 +164,8 @@ def _parse_args():
                    help='Call ``blaze.data`` on hidden files')
     p.add_argument('--yaml-dir', action='store_true',
                    help='Load path-based resources relative to yaml file directory.')
+    p.add_argument('--allow-dynamic-addition', action='store_true',
```

Comment:
It's good this is a long option name to make it less likely to trigger accidentally.